packages/block-library/src/index.native.js

```diff
@@ -88,6 +88,7 @@ export const coreBlocks = [
 	column,
 	cover,
 	embed,
+	group,
```

Comment:
Interesting that this was missing 😄

Reply:
Indeed. The value appears to only be used within test and the `core/missing` block.